GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/PurchaseController.cs

```diff
@@ -226,16 +226,16 @@ public PurchaseProcessingResult ProcessPurchase(PurchaseEventArgs args)
         return PurchaseProcessingResult.Complete;
 #endif
     }
-    
-    public static void ConfirmPendingPurchase(Product product, bool succes)
+
+    public static void ConfirmPendingPurchase(Product product, bool success)
```

Comment:
What does this bool variable do?

Reply:
This is written by Rafeal, I just changed a typo here. @RafaelHiguera

Reply:
Is a response from the server, telling the client if the verification of the purchase token was successful or not.

Reply:
Got it, please name it more contextual, e.g. purchaseVerifiedSuccess.

Reply:
Yep, good idea